src/index.html

```diff
@@ -1,13 +1,376 @@
+                <div class="tooth__image">
+                  <img src="./images/tooth.svg" alt="tooth">
+                </div>
+              </div>
+            </div>
+            <div class="tooth__button">
+              <a href="#" class="button__lern">LEARN MORE</a>
+            </div>
+            <div class="tooth__bottom">
+              <div class="tooth__bottom--position">
+                <img
+                  src="./images/tooth/tooth-icons/rabbit.svg"
+                  alt="rabbit"
+                  class="tooth__bottom__photo"
+                >
+                <p class="tooth__bottom__text">No synthetics and microplastics</p>
+              </div>
+              <div class="tooth__bottom--position">
+                <img
+                  src="./images/tooth/tooth-icons/recycle.svg"
+                  alt="recycle"
+                  class="tooth__bottom__photo"
+                >
+                <p class="tooth__bottom__text">product packaging can be recycled or reused</p>
+              </div>
+              <div class="tooth__bottom--position">
+                <img
+                  src="./images/tooth/tooth-icons/leaves.svg"
+                  alt="leaves"
+                  class="tooth__bottom__photo"
+                >
+                <p class="tooth__bottom__text">only natural ingredients</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="contact" id="contact">
+        <div class="container">
+          <div class="contact__content">
+            <div class="contact__filling">
+              <h2 class="contact__section-title section-title">contact us</h2>
+              <p class="credo">help </p>
+              <p class="contact__text">We will help you choose a product that suits you and answer your other questions</p>
+              <form
+                method="get"
+                class="contact__form form form--margin"
+                onsubmit="this.reset(); return false;"
+              >
+                <label>
+                  <input
+                    type="text"
+                    class="form__item"
+                    placeholder="Your name"
+                    autocomplete="off"
+                    required
+                  >
+                </label>
+                <label>
+                  <input
+                      type="email"
+                      class="form__item"
+                      placeholder="e-mail"
+                      autocomplete="off"
+                      required
+                    >
+                </label>
+                <label>
+                  <input
+                      type="tel"
+                      class="form__item form__phone"
+                      pattern="[0-9]{3}[0-9]{2}[0-9]{2}[0-9]{3}"
+                      placeholder="phone number"
+                      autocomplete="off"
+                      required
+                    >
```

Comment:
Should be only 2 spaces
```suggestion
                  <input
                    type="tel"
                    class="form__item form__phone"
                    pattern="[0-9]{3}[0-9]{2}[0-9]{2}[0-9]{3}"
                    placeholder="phone number"
                    autocomplete="off"
                    required
                  >
```